### DISCUSSION TITLE
why is not working ?

### DISCUSSION THREAD

**Discussion by SergeyArzumanyan** [community/community#112276] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

console.log("Hello, world!"

**Comment by RobertArustamyan** [comment, marked as answer]
You dont close Right )

**Comment by ghostinhershell** [comment]
Please see my response here: https://github.com/orgs/community/discussions/112279#discussioncomment-8765821
